ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastImportBlocksStep.java

```diff
@@ -75,7 +77,7 @@ public void accept(final List<BlockWithReceipts> blocksWithReceipts) {
     final long endTime = System.nanoTime();
 
     accumulatedTime += TimeUnit.MILLISECONDS.convert(endTime - startTime, TimeUnit.NANOSECONDS);
-    if (accumulatedTime > 10 * 1000L) {
+    if (accumulatedTime > TEN_SECONDS) {
```

Comment:
whyt not something like that ?  TimeUnit.SECONDS.toMillis(10)

Reply:
to avoid to recompute static value every time, and I think the constant name could easier to read

Reply:
ah I hadn't seen that you put it in static.